docs/index.md

```diff
@@ -1,11 +1,45 @@
 ---
 page_title: "MAAS Provider"
 description: |-
-  /* TODO: Add description of provider */
+    The Terraform Provider for MAAS.
+  
 ---
+<!-- "MAAS Terraform provider reference" -->
+If you wish to use MAAS with [Terraform](https://www.terraform.io/), there is a [provider avaiable](https://github.com/maas/terraform-provider-maas), supplied by a third party.  This article provides reference information about the data sources and resources available through the provider.  It does not attempt to explain the mechanics or usage of Terraform or offer any tutorial information related to this MAAS Terraform provider.
```

Comment:
```suggestion
```
I don't think this line is necessary since this file is for the landing page of provider's documentation on the Terraform registry e.g. https://registry.terraform.io/providers/hashicorp/aws/latest/docs 

@billwear @cpg1111 are you happy for me to take this line out?

Reply:
Seems like this is fine to remove.